chainerx_cc/chainerx/python/routines.cc

```diff
@@ -1002,13 +1003,46 @@ void InitChainerxPooling(pybind11::module& m) {
           py::arg("pad_mode") = "ignore");
 }
 
+void InitChainerxLoss(pybind11::module& m) {
+    m.def("mean_absolute_error",
+          [](const ArrayBodyPtr& x0, const ArrayBodyPtr& x1) { return MoveArrayBody(MeanAbsoluteError(Array{x0}, Array{x1})); },
+          py::arg("x0"),
+          py::arg("x1"));
```

Comment:
Could you start with 1 instead of 2 (similar to other ChainerX routines)?

Reply:
Instead of 0 right? `x1` and `x2`?

Reply:
Yes!